examples/using-local-plugins/README.md

```diff
@@ -16,4 +16,4 @@ This example uses a [local plugin](/docs/loading-plugins-from-your-local-plugins
 2. Process that data into Gatsby's node format
 3. Use the [`createNode` action](/docs/actions/#createNode) to add the data to Gatsby’s GraphQL layer
 
-The [`gatsby-node.js` file](https://github.com/jlengstorf/gatsby-with-unstructured-data/blob/using-gatsby-data-layer/plugins/gatsby-source-pokeapi/gatsby-node.js) of the local plugin includes detailed comments on the process.
+The [`gatsby-node.js` file](./plugins/gatsby-source-pokeapi/gatsby-node.js) of the local plugin includes detailed comments on the process.
```

Comment:
This is not an internal link, so should not be a relative path.

Reply:
Wait, should it be a relative path, though? Seems like a reader would expect to be linked to the `gatsby-node.js` file of this local plugin in this example, not the file in the external repo that these examples were pulled from.

Perhaps if it's important to leave a link to jlengstorf's repo, we should just do that in a separate section?

Reply:
You're correct. I missed the context! Good catch.